sdk-extensions/jaeger-remote-sampler/src/main/java/io/opentelemetry/sdk/extension/trace/jaeger/sampler/JaegerRemoteSampler.java

```diff
@@ -49,6 +57,9 @@ public final class JaegerRemoteSampler implements Sampler, Closeable {
     this.serviceName = serviceName != null ? serviceName : "";
     this.delegate = delegate;
     this.sampler = initialSampler;
+    this.resultAttributes =
+        Attributes.of(
+            SAMPLER_TYPE, TYPE, SAMPLER_PARAM, "level=service;" + initialSampler.getDescription());
```

Comment:
Honestly no. This is not part of any spec. I added this so that it adds more context to the decision.

Reply:
FYI, this is the code in Jaeger that checks for probabilistic sampling type in order to allow adaptive sampling to work.

https://github.com/jaegertracing/jaeger/blob/14c11365a85612c210eb0a526cab2a377a09b016/plugin/sampling/strategystore/adaptive/aggregator.go#L109

Reply:
I added this "level=service" more for debugability reason. It will help in identifying if the sampling was done done based on operation level policy or service level policy. It does not serve any purpose in adaptive sampling related metric aggregation at the collector.